src/commands/bot.ts

```diff
@@ -9,7 +9,7 @@ import { handlePing } from "../modules/subcommands/handlePing";
 
 export const bot: Command = {
   data: new SlashCommandBuilder()
-    .setName("bot")
+    .setName("eddiebot-breakout-status")
```

Comment:
I'm curious why the command name changes.

Reply:
`bot` seemed to generic

Reply:
IMHO, the command names *should* be generic (these are what the user has to type).

The description is where extra information can be displayed.

Reply:
Ok may be `network` would be better